### PR TITLE
[API][DOCS] custom path

### PR DESCRIPTION
**Pull request by AdamKasp**
| Q               | A
| --------------- | -----
| Branch?         | 1.9
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| License         | MIT

### CODE REVIEW

File: docs/api/unified_api/custom_paths.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Custom Paths in new Api
+=======================
+
+All path in new API have the same prefix structure: ``/api/v2/admin/`` or ``/api/v2/shop/``
+
+When you adding a new path to api resource configuration, you should remember to add also proper prefix.
```
Comment (arti0090) [inline docs/api/unified_api/custom_paths.rst:6]:
```suggestion
When you are adding a new path to api resource configuration, you should remember to add also proper prefix.
```

File: docs/api/unified_api/custom_paths.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+or you can add proper prefix for all path in chosen resource:
+
+.. code-block:: xml
+
+    <attribute name="route_prefix">shop</attribute>
+
+.. note::
+
+    In some situations you may need add path with custom structure, in this cases you should configure it in security file (``ApiBundle/test/config/security.yaml``)
```
Comment (arti0090) [inline docs/api/unified_api/custom_paths.rst:25]:
```suggestion
    In some situations you may need to add a path with custom structure, in this case you should configure it in security file (``ApiBundle/test/config/security.yaml``)
```
or 'those cases' in plural form. But looking at context I would prefer this case ( just like in suggestion)

File: docs/api/unified_api/custom_paths.rst [hunk 3]
```diff
@@ -0,0 +1,27 @@
+Custom Paths in new Api
+=======================
+
+All path in new API have the same prefix structure: ``/api/v2/admin/`` or ``/api/v2/shop/``
```
Comment (GSadee) [inline docs/api/unified_api/custom_paths.rst:4]:
```suggestion
All paths in new API have the same prefix structure: ``/api/v2/admin/`` or ``/api/v2/shop/``
```

File: docs/api/unified_api/custom_paths.rst [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+When you are adding a new path to api resource configuration, you should remember to add also proper prefix.
```
Comment (GSadee) [inline docs/api/unified_api/custom_paths.rst:6]:
```suggestion
When you are adding a new path to API resource configuration, you should remember to add also proper prefix.
```

File: docs/api/unified_api/custom_paths.rst [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+You can declare entire path for each operation (without ``/api/v2/`` this part is configured globally):
```
Comment (GSadee) [inline docs/api/unified_api/custom_paths.rst:8]:
```suggestion
You can declare the entire path for each operation (without ``/api/v2/`` as this part is configured globally):
```

File: docs/api/unified_api/custom_paths.rst [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+.. code-block:: xml
+
+    <collectionOperation name="admin_get">
+        <attribute name="method">GET</attribute>
+        <attribute name="path">admin/orders</attribute>
+    </collectionOperation>
+
+or you can add proper prefix for all path in chosen resource:
```
Comment (GSadee) [inline docs/api/unified_api/custom_paths.rst:17]:
```suggestion
or you can add a proper prefix for all paths in the chosen resource:
```

File: docs/api/unified_api/custom_paths.rst [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+.. code-block:: xml
+
+    <attribute name="route_prefix">shop</attribute>
+
+.. note::
+
+    In some situations you may need to add a path with custom structure, in this case you should configure it in security file (``ApiBundle/test/config/security.yaml``)
```
Comment (GSadee) [inline docs/api/unified_api/custom_paths.rst:25]:
```suggestion
    In some situations, you may need to add a path with a custom structure, in this case, you will probably need to configure also the appropriate access in the security file (``ApiBundle/test/config/security.yaml``)
```

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Adam! :tada:

**Pull request merged**